AFQ/api/participant.py

```diff
@@ -96,14 +96,13 @@ def __init__(self,
                 "did you mean tracking_params ?"))
 
         self.logger = logging.getLogger('AFQ')
-        self.output_dir = output_dir
+        self.bids_info = _bids_info
 
         self.kwargs = dict(
-            dwi_path=dwi_data_file,
-            bval=bval_file,
-            bvec=bvec_file,
-            results_dir=output_dir,
-            bids_info=_bids_info,
```

Comment:
@36000 : was there a reason for these substitutions? That is, that `dwi_data_file` becomes `dwi_path`, and so on? Would there be a cost associated with this change so that internal kwargs and external names are the same?